lib/components/SInputFileUpload.vue

```diff
@@ -50,28 +79,46 @@ const { t } = useTrans({
   }
 })
 
+const dropZoneEl = ref<HTMLDivElement | null>(null)
+
+const { isOverDropZone } = useDropZone(dropZoneEl, {
+  multiple: true,
+  onDrop: (files) => onDrop(files)
+})
+
 const _value = computed(() => {
   return props.modelValue !== undefined
     ? props.modelValue
-    : props.value !== undefined ? props.value : []
+    : props.value !== undefined ? props.value : [] as unknown as ModelValue<T>[]
```

Comment:
as unknown assertion is probably not needed, [] should be directly castable to ModelValue<T>[]

Reply:
You are right. Fixed! I thought it was showing error but probably fixed when I changed something 🫠